core/src/main/java/cucumber/api/event/Event.java

```diff
@@ -8,7 +8,4 @@
     Comparator<Event> CANONICAL_ORDER = new CanonicalEventOrder();
 
     Long getTimeStamp();
-
-    Thread getThread();
```

Comment:
I've removed this and replaced it with a interface based guarantee that  `Thread.currentThread()` will return something useful when used in a formatter.

Reply:
The reason being that the grantee is needed for some tooling that uses thread-local internally to track concurrent execution.

Reply:
Fair enough. As long as the correct data is still being output I really don’t mind. 

Have you ran a set of features multi threaded to ensure there are still multiple groups in the output?  This is something I cannot put in the test as last time they kept failing in Travis as it wasn’t running with multiple threads

Reply:
Update, verified that the output is all good